MoneyTransferWorker/Activities.cs

```diff
@@ -0,0 +1,50 @@
+namespace MoneyTransferProject;
+using Temporalio.Activities;
+
+public class BankingActivities 
+{
+    [Activity]
+    public async Task<string> WithdrawAsync(PaymentDetails details)
+    {
+        var bankService = new BankingService("bank1.example.com");
+        Console.WriteLine($"Withdrawing ${details.Amount} from account {details.SourceAccount}.");
+        try
+        {
+            return await bankService.Withdraw(details.SourceAccount, details.Amount, details.ReferenceID);
+        }
+        catch (Exception ex)
+        {
+            throw new ApplicationException("Withdrawal failed", ex);
+        }
```

Comment:
You don't _have_ to catch-and-wrap here if you don't want, but it's ok to do to provide a common outer message obviously